### ISSUE TITLE
`autoAck: true` causes an non-descriptive error when not acked.

### ISSUE THREAD

**Issue by alexgorbatchev** [tessel/rf-nrf24#15]
`autoAck: true` causes an non-descriptive error when not acked. I feel it could be better, took a while to figure out what the issue was.

```Packet timeout, transmit queue flushed.```

**Comment by natevw** [comment]
Good point. How about "Packet not acknowledged within autoRetransmit window. Transmit queue has been flushed!"?

**Comment by alexgorbatchev** [comment]
I would suggest a bit more info, like "Packet not acknowledged within autoRetransmit window because `autoAck` is set to `true`. Transmit queue has been flushed!"
